src/main/java/Baraleous.java

```diff
@@ -0,0 +1,160 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Baraleous {
+    public static void main(String[] args) {
+        String logo = "Hello! I'm Baraleous XIV!\n" +
+                "What can I do for you today?";
+        printMessage(logo, false);
+        Scanner scanner = new Scanner(System.in);
+        TaskList taskList = new TaskList();
+        commandLoop: while(true){
```

Comment:
Adding to the above comment, space in between `(true)` and `{` as well:

```suggestion
        commandLoop: while (true) {
```